docs/topics/installing-cli-tool.adoc

```diff
@@ -20,7 +20,33 @@ However, if you want to analyze applications in languages other than Java or, fo
 
 .Prerequisites
 
-* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from `registry.redhat.io`, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
+The following are the prerequisites for the {ProductFullName} CLI installation:
+
+* Java Development Kit (JDK) is installed. {ProductShortName} supports the following JDKs:
+
+** OpenJDK 17 or later
+** Oracle JDK 17 or later
+** Eclipse Temurin™ JDK 17 or later
+
+* 8 GB RAM is required for the {ProductShortName} {CLIName}.
+
+// Let us flag to MTA-QE channel
+// * macOS installation: the value of `maxproc` must be `2048` or greater.
+
+* Linux installation requires the `java-<JDK-version>-devel` package and not only the `java--<JDK-version>` package.
++
+For example, to install OpenJDK 17 on {op-system-base-full}, you would run:
++
+[source,terminal]
+----
+sudo dnf install java-17-openjdk java-17-openjdk-devel
+----
```

Comment:
@anarnold97 regarding this whole "Linux installation" part above, I think I would remove it entirely. It should be known to the developer how to properly install a JDK distribution. The definition of "JDK" itself implies having all the development tools.

What happened to @ibragins was that he had installed only the "JRE" (Java Runtime Environment), which in the particular case of the Fedora package manager (not all Linux distros) is called `java-17-openjdk`.

If you don't want to remove it entirely, I would maybe switch it to an asterisk saying something like
> * Please note that MTA requires a fully installed JDK. Having a JRE only will cause issues during analysis.
and leave it to the developer to figure out how to install it for his particular OS.

Reply:
This is not correct @jmle , I installed OpenJDK, not JRE. The fact is that running `sudo dnf install java-17-openjdk` on Fedora doesn't install a devel package which actually caused the problem and is the reason of this conversation